src/aggregate/aggregate_request.c

```diff
@@ -356,6 +356,25 @@ static int handleCommonArgs(ParseAggPlanContext *papCtx, ArgsCursor *ac, QueryEr
       BM25STD_TANH_FACTOR_MIN, BM25STD_TANH_FACTOR_MAX);
       return ARG_ERROR;
     }
+  } else if ((*papCtx->reqflags & QEXEC_F_INTERNAL) && AC_AdvanceIfMatch(ac, "_SLOTS")) {
```

Comment:
can we name it _SLOT_RANGES? It is more clear than just slots that can be thought as individual ones